### PR TITLE
chore: bug: fix: CRW-1995 according to https://www.npmjs.com/package/keytar need to install libsecret-devel on RHEL systems

### PR DESCRIPTION
**Pull request by nickboldt**
### What does this PR do?

chore: bug: fix: CRW-1995 according to https://www.npmjs.com/package/keytar need to install libsecret-devel on RHEL systems 
chore: bug: fix: put back deleted line that creates theia-source-code.tgz as we need that in downstream bootstrapping

Change-Id: I9b3ed273062cff2a4a6d33ea398131f25dc1f085
Signed-off-by: nickboldt <nboldt@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
N/A (or see commit message above for issue number)

### How to test this PR?
N/A

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works, if one exists](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections What issues does this PR fix or reference and How to test this PR completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: dockerfiles/theia/docker/ubi8/builder-clone-theia.dockerfile [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # Clone theia and keep source code in home
 RUN git clone --branch ${GIT_BRANCH_NAME} --single-branch https://github.com/${THEIA_GITHUB_REPO} ${HOME}/theia-source-code && \
     cd ${HOME}/theia-source-code && git checkout ${THEIA_COMMIT_SHA}
+RUN cd ${HOME} && tar zcf ${HOME}/theia-source-code.tgz theia-source-code
```
Comment (benoitf) [inline dockerfiles/theia/docker/ubi8/builder-clone-theia.dockerfile:4]: why do we need that ?
https://github.com/redhat-developer/codeready-workspaces-theia/blob/crw-2-rhel-8/dockerfiles/theia/Dockerfile#L36

Comment (benoitf) [reply dockerfiles/theia/docker/ubi8/builder-clone-theia.dockerfile:4]: I mean why did you remove it from https://github.com/redhat-developer/codeready-workspaces-theia/commit/8784e951a715a84cbdcc6037af91c44d1c935780#diff-aabad8f4a3b5d17a3038533a360278704d1160ac9d0a3ad4ff66d588f20c399d

Comment (benoitf) [reply dockerfiles/theia/docker/ubi8/builder-clone-theia.dockerfile:4]: please don't do any tgz there, we don't need it

Comment (nickboldt) [reply dockerfiles/theia/docker/ubi8/builder-clone-theia.dockerfile]: I didn't remove it ... it was removed upstream and then that change was synced to downstream :) But it's OK, we can work around this downstream by overriding .dockerfile stubs.

**Review by benoitf — APPROVED** [review]

**Comment by nickboldt** [comment]
```
No match for argument: libsecret-devel
Error: Unable to find a match: libsecret-devel
```

Well, crap.

**Pull request closed without merging**